main_sync_cffi.py

```
@@ -0,0 +1,94 @@
+        resp_headers.pop("content-length", None)  # Remove as we're streaming
+        resp_headers.pop("content-encoding", None)  # Remove encoding headers
+        
+        return StreamingResponse(
+            stream_response(),
+            status_code=response.status_code,
+            headers=resp_headers,
+            media_type=resp_headers.get("content-type", "text/event-stream")
+        )
+        
+    except Exception as e:
+        logger.exception(f"Error proxying request: {e}")
```

Ruff (0.12.2)

Redundant exception object included in `logging.exception` call

(TRY401)